### DISCUSSION TITLE
Change Background color on condition

### DISCUSSION THREAD

**Discussion by DevrayePatil** [Budibase/budibase#7471] [Help] [answered]
I want to change background color based on the condition 
How can I achieve this?

![image](https://user-images.githubusercontent.com/54846160/186877160-100c98c6-7070-4f71-a110-a74381c34848.png)

**Comment by R2bEEaton** [comment, marked as answer]
Hi @DevrayePatil,

If you use the Tag component, you can use Conditions to update the Color setting. Here you can see I used this to change it from orange to yellow.
![image](https://user-images.githubusercontent.com/34921506/186881139-87bf0961-ad70-4634-b736-026491ac2e05.png)

For more complicated CSS changes based on a condition, see this PR #7407, which stems from #5818. Looks like a solution is in the works.

Enjoy!

**Reply by DevrayePatil** [reply]
Ok thank you

**Reply by DevrayePatil** [reply]
I had another question there is no option for css as shown in issue no. #7407 for which component we can apply custom css using conditional UI

**Reply by R2bEEaton** [reply]
The update isn't out yet.
